data/scrape/lib/README.md

```diff
@@ -52,7 +52,11 @@ The following steps can be used to run the big scrape procedure for a subset:
     line arguments for desired languages. Note: languages must be in their ISO
     designation and argument string must delineate with comma, semicolon, or
     space. E.g. To target only Lithuanian and Spanish:
-    `./scrape.py --restriction='lit; spa'`
+    `./scrape.py --restriction='lit; spa'`.
+    To target all but one or more specific languages, apply the `--exclude` flag,
+    e.g., to exclude Lithuanian and Spanish:
+    `./scrape.py --exclude='lit; spa'`.
+    Only one, but not both, of `--restriction` and `--exclude` can be applied.
```

Comment:
I'd prefer to keep things simple to the user / us by imposing mutual exclusion here between `--restriction` and `--exclude`, although as you'll see from my changes the underlying code technically allows their concurrent use.

Reply:
Fine with me.

My question is why not use `nargs='*'` so you can do `--restriction lit --restriction spa` or something like that? Thoughts?

Reply:
To get updated data for the upcoming SIGMORPHON G2P task, I was able to do something like `--restriction='<code> <code> <code> ...'` for a whole bunch of languages, rather than the otherwise unwieldy `--restriction <code>` being repeated for however many languages I needed to handle.